app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -293,14 +299,25 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
                     }
                 }
         );
+        findInPageQuery.setOnClickListener(this);
+        findInPageQuery.setOnEditorActionListener(new TextView.OnEditorActionListener() {
+            @Override
+            public boolean onEditorAction(TextView textView, int i, KeyEvent keyEvent) {
+                if (i == EditorInfo.IME_ACTION_DONE) {
```

Comment:
Let's use something that is more descriptive than ``i``. The interface of OnEditorActionListener uses `actionId`.